client/src/js/users/components/__tests__/Create.test.js

```diff
@@ -1,71 +1,182 @@
+jest.mock("connected-react-router");
+
 import * as actions from "../../actions";
-import CreateUserContainer, { CreateUser } from "../Create";
+import CreateUserContainer, { CreateUser, mapStateToProps, mapDispatchToProps } from "../Create";
+import { push } from "connected-react-router";
 
 describe("<CreateUser />", () => {
     let props;
-    let wrapper;
+    let state;
+    let hasError;
 
-    it("renders correctly", () => {
+    beforeEach(() => {
         props = {
-            show: true,
-            pending: false,
-            minimumPasswordLength: 8,
-            error: "",
-            onCreate: sinon.spy(),
-            onHide: sinon.spy(),
-            onClearError: sinon.spy()
+            error: "foo",
+            onClearError: jest.fn(),
+            onCreate: jest.fn()
         };
 
-        wrapper = shallow(<CreateUser {...props} />);
+        state = {
+            confirm: "",
+            errorConfirm: "",
+            errorPassword: "",
+            errorUserId: "foo",
+            forceReset: false,
+            password: "",
+            userId: ""
+        };
+    });
+
+    it("should render", () => {
+        const wrapper = shallow(<CreateUser {...props} />);
         expect(wrapper).toMatchSnapshot();
     });
 
-    describe("Dispatch Functions", () => {
-        const initialState = {
-            router: { location: { state: { createUser: true } } },
-            users: { createPending: false },
-            settings: { data: { minimum_password_length: 8 } },
-            errors: null
+    it("should call handleChange when InputError is changed", () => {
+        let name, value, error;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variables within the same declaration block should be sorted alphabetically. (sort-vars)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4652072)